databricks_cli/stack/api.py

```diff
@@ -139,6 +162,84 @@ def deploy_config(self, stack_config, stack_status=None, **kwargs):
 
         return new_stack_status
 
+    def download_config(self, stack_config, **kwargs):
```

Comment:
Minor: 
1) Nomenclature is a bit off. It sounds as if you're downloading the config. 
2) Consider making it private.

Reply:
i'm thinking of actually exposing this in the future and putting the code in `download` in the cli.py file.

Reply:
The `config` part is confusing because the config file is already a local file. Maybe "_download_resources"?

Reply:
Now it seems unnecessary to break `download` and `download_config`? Now both functions are very thin and it is hard to tell the differences. Maybe they can be combined into one method?

Reply:
Thinking of making another PR to actually move what is in `download` and anything related to reading or saving JSON's to `cli.py` and make `download_config` the main entry endpoint of the API. Let me know what you guys think of this.